alert-common/src/main/java/com/synopsys/integration/alert/common/message/model/ContentKey.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * alert-common
+ *
+ * Copyright (c) 2019 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.common.message.model;
+
+import java.util.List;
+
+import org.apache.commons.lang3.StringUtils;
+
+import com.synopsys.integration.alert.common.rest.model.AlertSerializableModel;
+
+public class ContentKey extends AlertSerializableModel {
+    private static final String KEY_SEPARATOR = "_";
```

Comment:
This is the same KEY_SEPARATOR as the one in ComponentKeys, could we have them share the same separator?